core/domain/config_domain.py

```diff
@@ -30,6 +30,24 @@
 
 CMD_CHANGE_PROPERTY_VALUE = 'change_property_value'
 
+LIST_OF_FEATURED_TRANSLATION_LANGUAGES_DICTS_SCHEMA = {
+    'type': 'list',
+    'items': {
+        'type': 'dict',
+        'properties': [{
+            'name': 'language_code',
+            'schema': {
+                'type': 'unicode'
+            },
```

Comment:
should we add a validator `is_valid_translation_lanaguage_code`?

Reply:
Done